### ISSUE TITLE
Fix verbalization of Flutter logos in search results page

### ISSUE THREAD

**Issue by atsansone** [flutter/website#10540]
From https://buganizer.corp.google.com/issues/337307320

### Observed Results

Navigating through the page specifically on the flutter buttons, listen to how are verbalized by SR.

* [Screencast VoiceOver](http://screencast/cast/NjE5MDcwODMzNDUyNjQ2NHxmY2M5MDdlMy1iOA)
* [Screencast ChromeVox](http://screencast/cast/NTEyNzY3MDg0NDA5NjUxMnw3NmY0MzA2Mi0wNA)

### Expected Results

Ensure that the flutter button should have the correct label that is a button opening a new window/tab.

Example: Flutter button that opens the use of memory in a new window/tab.

### User Impact

Screen reader and Cognitive users are affected because those buttons are announced as images and never indicate that open a new window/tab.

### Steps to Reproduce

1. Enabled VoiceOver using <kbd>Command</kbd> + <kbd>F5</kbd> also ChromeVox using <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Z</kbd>
1. Open the following page: https://docs.flutter.dev/search?q=devtools
1. Continue the linear navigation using the <kbd>Tab</kbd> key and reach the flutter buttons.
1. Listen that they are announced as thumbnail images and never notified that open a new window/tab.

### GAR Assessment

[Flutter Docs](https://a11ytracker.corp.google.com/products/c2fdcf18-53d2-475d-b8b0-e5dedf8cb900?e=ProductLabels)

### GAR Details

[1.11. Is the purpose of each link, control, and button clear?](go/GAR-web#purpose-link-control-button)

### Environment Details

Machine: MAC/ChromeBook
Browser: Chrome Version 120.0.6099.272 (Official Build) (64-bit)
Screen reader: VoiceOver / ChromeVox.
Bug filling time: 15 mins.

**Comment by parlough** [comment]
These search results are generated, styled, and managed by the Google Custom Search setup we use.

Since we don't control that, I'm going to close this in favor of an umbrella issue https://github.com/flutter/website/issues/11800 which tracks implementing a new experience that we can properly integrate and style. Thanks!
